docs/examples/expressions/cats.txt

```diff
@@ -0,0 +1,10 @@
+kitten/cat_1.jpg 1
```

Comment:
Curious, is it possible to just let the reader get all the files in the subfolders?

Reply:
It assumes some structure of `root_dir/class_dir`. And I want two explicit batches that are separate (cats vs dogs).